modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/web/socket/WebSocketManager.java

```diff
@@ -338,6 +349,11 @@ private void handleBrowserEvents(WebSocketSession ws, TextMessage msg) throws IO
         WebSocketEvent evt = fromJson(msg.getPayload(), WebSocketEvent.class);
 
         switch (evt.getEventType()) {
+            case ADMIN_ANNOUNCEMENT:
```

Comment:
Set announcement don't have permission check. I think will be better to return this logic to REST

Reply:
Fixed